recipe/meta.yaml

```diff
@@ -152,10 +153,35 @@ outputs:
         - PyQt5.QtWidgets
         - PyQt5.QtXml
         - PyQt5.QtXmlPatterns
+        - PyQt5.QtWebKit
       commands:
         - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python pyqt_test.py'  # [linux and build_platform == target_platform]
         - pyuic5 --version  # [not win]
 
+
+  - name: pyqtwebkit
+    version: {{ version }}
+    script: build-pyqtwebkit.sh  # [not win]
+    script: bld-pyqtwebkit.bat  # [win]
```

Comment:
I think this file does not exist

Reply:
Ugh, I pushed the wrong branch, hang on.